settings/R/MultiSettings.R

```diff
@@ -0,0 +1,195 @@
+##' Create a PEcAn MultiSettings object
+##'
+##' 
+##' @title Create a PEcAn MultiSettings object
+##' @param 
+##' @return
+##' @export
+##' @author Ryan Kelly
+MultiSettings <- function(...) {
+  result <- list(...)
+
+  if(length(result) == 1) {
+    if(is.MultiSettings(result[[1]])) {
+      return(result[[1]])
+    } else if(is.list(result[[1]]) && all(sapply(result[[1]], is.Settings))) {
+      result <- result[[1]]
+    }
+  }
+  
+  if(!all(sapply(result, is.Settings))) {
+    stop("MultiSettings can only be made from Setting, MultiSettings, or a list of Settings")
+  }
+
+  if(length(result) > 0)
+    names(result) <- paste("settings", seq_along(result), sep=".")
```

Comment:
Don't know the details of what you've implemented, but do you really want to overwrite any names that have already been defined, or do you only want to do this if names are absent?  E.g. If I have a list of Settings that I want to turn into a multi-site, and I'd named the list elements in a way that was useful to my analysis, does that information need to be dropped for MultiSettings to work?